src/box/lua/config/utils/schema.lua

```diff
@@ -1492,6 +1492,88 @@ end
 
 -- }}} <schema object>:pairs()
 
+-- {{{ <schema object>:jsonschema()
+
+local json_scalars = {
+    string = {jsonschema = {type = 'string'}},
+    number = {jsonschema = {type = 'number'}},
+    ['string, number'] = {jsonschema = {type = {'string', 'number'}}},
+    ['number, string'] = {jsonschema = {type = {'string', 'number'}}},
+    integer = {jsonschema = {type = 'integer'}},
+    boolean = {jsonschema = {type = 'boolean'}},
+    any = {jsonschema = {}}
+}
+
+-- This function sets common fields such as `description`, `default`, and `enum`
+-- for a given schema object. These fields are typical in JSON schemas.
+local function set_common_fields(res, schema)
```

Comment:
I would prefix it with `jsonschema_` or include it into the function name at least (like `set_common_jsonschema_fields()`).

Reply:
Renamed.